src/content/app/genome-browser/components/drawer/components/sequence-view/DrawerSequenceView.tsx

```diff
@@ -91,12 +94,21 @@ const DrawerSequenceView = (props: Props) => {
 
   const canHaveReverseComplement = selectedSequenceType === 'genomic';
 
+  const onShowHideClick = () => {
+    toggleSequenceVisibility();
+
+    // Track only when it is opened
+    if (!isExpanded) {
+      trackDrawerSequenceViewed(selectedSequenceType);
```

Comment:
When the drawer sequence opens, the sequence type will always be "genomic" (unless you have switched to a different sequence type, closed the sequence view, and opened it again). In view of this, what is the value of reporting selected sequence type? Was the intention to report selected sequence types when you are switching between them?

Reply:
I've updated it so that it initially reports "genomic" and then sends additional events with different types when the user changes the radio.